lib/Mojo/Util.pm

```diff
@@ -282,8 +282,8 @@ sub tablify {
     }
   }
 
-  my $format = join '  ', map({"\%-${_}s"} @spec[0 .. $#spec - 1]), '%s';
-  return join '', map { sprintf "$format\n", @$_ } @$rows;
+  my @formats = (map({"\%-${_}s"} @spec[0 .. $#spec - 1]),'%s');
+  return join '', map { sprintf join(' ', @formats[0..scalar @$_ - 1]) . "\n", @$_ } @$rows;
```

Comment:
Columns should be joined on 2 spaces, this appears to be the cause of test failures. `@formats[0..scalar @$_ - 1]` can be shortened to `@formats[0..$#$_]`